FindOpenSplice.cmake

```diff
@@ -71,17 +75,32 @@ function(OpenSplice_configure_targets config homeDir csEnabled)
             "${homeDir}/include/dcps/C++/SACPP")
         set_property(TARGET "OpenSplice::isocpp2" PROPERTY INTERFACE_LINK_LIBRARIES
             "OpenSplice::ddskernel")
+          # These are needed for dlopen w default ospl.xml: OpenSplice::ddsi2;OpenSplice::spliced;OpenSplice::durability
```

Comment:
This comment seems somewhat misplaced, and the two empty lines below it are unnecessary, IMO. Was it supposed to be here?